Dockerfile

```
@@ -10,7 +10,7 @@ RUN go mod download
 # build
 COPY . .
```

this means we're now copying the Rust validator code into the build container for the Go `proxy-init` container. we should probably just copy the proxy-init code, so that we can cache the Go code even when the Rust code changes?

i think we might be able to just change this to
```suggestion
COPY ./proxy-init .
COPY ./pkg .
```
(unless i've overlooked some other file we also need to copy over?)